### ISSUE TITLE
Ability to pipe commands in tooling

### ISSUE THREAD

**Issue by neclimdul** [lando/lando#1918]
Recreating #1832 because stale bot closed it. This is _still_ broken and completely breaks the documented workflow requiring manual interaction to _setup_ a lando instance and do standard tooling processes.

These issues are closed but it is still not possible to use pipes in tooling or other commands
#951 #967

Like this redirection

name: landobrokenpipe
recipe: custom
config:
  webroot: .
services:
  test:
    type: node:8
tooling:
  compile:
    service: test
    cmd: 
       - echo "Do some compile steps..."
       - sed ERROR compile.log > errors.txt
or this pipe I'd like to use to test pecl yaml libraries in d8.

name: d8
recipe: drupal8

services:
  appserver:
    run_as_root:
      - "apt-get update"
      - "apt-get install -y libyaml-dev"
      - "printf \"\n\" | pecl install yaml-2.0.4"

**Comment by neclimdul** [comment]
.

**Comment by Peter-WAAT** [comment]
I'm suffering from this too. Appending to a file doesn't work with `>>` (a.k.a. redirection)...

**Comment by pirog** [comment]
@neclimdul i am not able to replicate this. We make fairly extensive usage of pipe and other special symbols in our tests

* For build steps: https://github.com/lando/lando/blob/master/examples/services/.lando.yml
* For tooling:  https://github.com/lando/lando/blob/master/examples/tooling/.lando.yml#L11

I'd encourage you to spin up those examples against the latest Lando and go through the verification steps in the README and report back. This might help us figure out whats different about your setup that is causing this to not work.

**Comment by smutlord** [comment]
@pirog queued up another func test for this
https://github.com/lando/lando/commit/e053d2da90fce6da8b4eeb49c43f34dc86e057df

**Comment by neclimdul** [comment]
Just getting around to actually testing this. I figured it was fixed and this was actually closed so I hadn't looked at it. My bad....

I think i figured out the problem I don't know if it is _actually_ a bug anymore though maybe it is. The yaml handling is interesting at least. See the example at the end of this comment.

I'm in the habit of quoting complicated strings in yaml to make sure they're parsed as expected by yaml parsers but somehow in this case it is leading to lando treating the commands differently. Its been a while since I've read the yaml spec but that doesn't _seem_ like these two lines should even be different to lando by the time it see's them but somehow it is. Its even more confusing that somehow the pipe is both triggering pecl to run but also somehow getting into the pecl arguments. Really weird.

Anyways, at least there's this comment for me to find google searching the next time I can't figure this out :-D


```
$ lando version
v3.0.0-rrc.3
$ lando yaml
<a bunch of pecl build logging>
$ lando yaml2
pecl: 1: printf: usage: printf format [arg ...]
pecl: 2: pecl: Syntax error: "|" unexpected
$ tail -n 8 .lando.yml
  yaml:
    service: appserver
    cmd:
      - printf \"\n\" | pecl install yaml-2.0.4
  yaml2:
    service: appserver
    cmd:
      - "printf \"\n\" | pecl install yaml-2.0.4"

```
